src/linefunctiondata.cc

```diff
@@ -91,55 +91,59 @@ JacPropMatType select_derivativeLineShape(const String& var,
 std::istream& LineShape::from_artscat4(std::istream& is,
                                        Model& m,
                                        const QuantumIdentifier& qid) {
-  // Special case when self is part of this
-  auto self_in_list = LegacyPressureBroadeningData::self_listed(
-      qid, LegacyPressureBroadeningData::TypePB::PB_PLANETARY_BROADENING);
-  auto i = self_in_list ? 1 : 0;
-
   // Set or reset variables
   m.mtype = Type::VP;
-  m.mself = bool(i);
+  m.mself = true;
   m.mbath = false;
-  m.mdata = std::vector<SingleSpeciesModel>(6 + i);
-  m.mspecies = ArrayOfSpeciesTag(6 + i);
+  m.mdata = std::vector<SingleSpeciesModel>(7);
+  m.mspecies = ArrayOfSpeciesTag(7);
 
   // Set species
-  m.mspecies[i] = SpeciesTag(String("N2"));
-  m.mspecies[i + 1] = SpeciesTag(String("O2"));
-  m.mspecies[i + 2] = SpeciesTag(String("H2O"));
-  m.mspecies[i + 3] = SpeciesTag(String("CO2"));
-  m.mspecies[i + 4] = SpeciesTag(String("H2"));
-  m.mspecies[i + 5] = SpeciesTag(String("He"));
+  m.mspecies[1] = SpeciesTag("N2");
```

Comment:
Is it correct that element 0 is not set?

Reply:
Yes.  Element "0" is "SELF" broadening in ARTSCAT-4.  When mself is true, then element "0" is not named in LineShape::Model.  The same for element "-1" when "AIR" broadening mbath is true.